test/2_create_test_app_namespace.sh

```diff
@@ -33,7 +33,7 @@ if [[ "$PLATFORM" = "openshift" ]]; then
     TEST_DIR="config/openshift"
 fi
 
-./$TEST_DIR/test-app-conjur-authenticator-role-binding.sh.yml | $cli_with_timeout "create -f -"
+wait_for_it 600  "./$TEST_DIR/test-app-conjur-authenticator-role-binding.sh.yml | $cli_without_timeout create -f -"
```

Comment:
Did this fail on retries because the input could only be read once?

Reply:
it read the yml file and than run the retry only on  create command, after it failed once it tried to create with no yml file